### ISSUE TITLE
EPP: contact_update "no errors" on contact with missing ident data

### ISSUE THREAD

**Issue by vohmar** [internetee/registry#179]
Updating contact with missing ident data generates unhandled error condition

```
<result code="1">
  <msg lang="en">handle_errors was executed when there were actually no errors</msg>
</result>
```

sample contact_info ident

```
    <extension>
      <eis:extdata xmlns:eis="https://epp.tld.ee/schema/eis-1.0.xsd">
        <eis:ident type="" cc=""/>
      </eis:extdata>
    </extension>
```

failure on update of such contact is OK, and the only solution for the registrar here is to create a new contact object, but correct error handling is necessary. An expected error message 

`2306    "Parameter value policy error. Update of ident data not allowed [ident]"

**Comment by artur-beljajev** [comment]
Please provide steps to reproduce

**Comment by vohmar** [comment]
1. create/find a contact object with missing ident data
2. update that contact object's ident data
  1.   if you set ident cc to EE the result will be internal error
  2.   if you set ident cc to something else (ie CH) the result will be the message in ticket description

in any case the result should be handeled correctly

**Comment by vohmar** [comment]
@artur-beljajev cannot find anything to test here, will move the ticket back to todo list

**Comment by artur-beljajev** [comment]
@vohmar Check if we could close this one (see referenced issue)

**Comment by vohmar** [comment]
preoblem looks to be solved with ticket #569
